packages/create-app/templates/default-app/packages/backend/Dockerfile

```diff
@@ -43,7 +50,7 @@ COPY --chown=node:node yarn.lock package.json packages/backend/dist/skeleton.tar
 RUN tar xzf skeleton.tar.gz && rm skeleton.tar.gz
 
 RUN --mount=type=cache,target=/home/node/.cache/yarn,sharing=locked,uid=1000,gid=1000 \
-    yarn install --frozen-lockfile --production --network-timeout 300000
+    yarn workspaces focus --all --production
```

Comment:
I took what we have done with the Demo site and what the Yarn 3 tutorial had. I can't link directly to it but this is the relevant paragraph:

> The `--production` flag to yarn install has been removed in Yarn 3, instead you need to use `yarn workspaces focus --all --production` to avoid installing development dependencies in your production deployment. A tradeoff of this is that yarn workspaces focus does not support the `--immutable` flag.

Reply:
I am wondering if I should add the `&& rm -rf "$(yarn cache clean)"` as well. 🤔